go/vt/vtgate/planbuilder/horizon_planning.go

```diff
@@ -381,7 +381,13 @@ func generateAggregateParams(aggrs []operators.Aggr, aggrParamOffsets [][]offset
 					aggrExpr = &sqlparser.BinaryExpr{
 						Operator: sqlparser.MultOp,
 						Left:     aggrExpr,
-						Right:    curr,
+						Right: &sqlparser.FuncExpr{
+							Name: sqlparser.NewIdentifierCI("coalesce"),
+							Exprs: sqlparser.SelectExprs{
+								&sqlparser.AliasedExpr{Expr: curr},
+								&sqlparser.AliasedExpr{Expr: sqlparser.NewIntLiteral("1")},
+							},
+						},
```

Comment:
To me this looks like something we would only need for outer joins, right? Is is worth adding that check? Most of the planner tests that are there in the file are doing inner joins, could you add one for left-join too?

Reply:
It is non-trivial to track outer joins here. I will add a test for left-join if it does not exist.